raiden-ts/src/channels/reducer.ts

```diff
@@ -216,7 +216,8 @@ const pendingTxs: Reducer<RaidenState['pendingTxs'], RaidenAction> = (
  * name of the reducer. channels root reducer instead must be handled the complete state instead,
  * so it compose the output with each key/nested/combined state.
  */
-export const channelsReducer = partialCombineReducers(
+const channelsReducer = partialCombineReducers(
   { blockNumber, tokens, channels, pendingTxs },
   initialState,
 );
+export default channelsReducer;
```

Comment:
Any specific reason you decided to go with the default export now?

Reply:
Mostly for consistency with other modules which export a single value. Also, on renaming of the module, I had to rename the property on the imports as well (`pathReducer` to `servicesReducer`), while as it was the only function exported, it didn't matter, only its location on the import, so changing it also makes one less place to rename on future refactors (which I hope won't help, but just to be consistent).